andy_api/api/intent_processing/choose_side.py

```diff
@@ -57,9 +59,12 @@ def handle(session_id, intent_model):
         set_game_started(session_id)
         set_chosen_side(session_id, user_side)
 
+        starting_board_str = chess.Board().board_fen
+        demo_board_str = "r2qk2r/pb4pp/1n2Pb2/2B2Q2/p1p5/2P5/2B2PPP/RN2R1K1 w - - 1 0"
```

Comment:
`demo_board_str` should be put inside of a constant toward the top of the file. We should also set `starting_board_str` to `demo_board_str` if the environment variable `DEMO_MODE` is True.